omero/users/cli/chown.txt

```diff
@@ -0,0 +1,176 @@
+
+Examples
+^^^^^^^^
+
+Basic transfer of ownership
+===========================
+
+::
+
+    $ bin/omero chown 5 OriginalFile:101
+    $ bin/omero chown User:5 Project:51
+    $ bin/omero chown Experimenter:5 Project:51
+    $ bin/omero chown jane Project:51
+
+In the first line, the ownership of original file with ID 101 will be
+transferred to the user with ID 5. In the second and third, the ownership
+of project 51 will be transferred including any datasets
+inside only that project and any images that are contained within transferred
+datasets only, as long as all the mentioned objects (project, datasets and
+images) are originally owned by one user. If user 5 is named 'jane' then the
+last line will have the same effect as the previous two. Note that any linked
+annotations will be transferred depending on the permission level of the group
+in which the data and users are in.
+
+Transferring multiple objects
+=============================
+
+Multiple objects can be specified with each type being followed by an ID
+or a comma-separated list of IDs. The order of objects or IDs is not
+significant, thus all three calls below are identical in transferring
+ownership of project 51 and datasets 53 and 54 to user 5.
+::
+
+    $ bin/omero chown 5 Project:51 Dataset:53,54
+    $ bin/omero chown 5 Dataset:54,53 Project:51
+    $ bin/omero chown 5 Dataset:53 Project:51 Dataset:54
+
+To transfer a number of objects with sequentially numbered IDs a hyphen can
+be used to specify an ID range. This form can also be mixed with
+comma-separated IDs.
+::
+
+    $ bin/omero chown 5 Project:51 Dataset:53-56
+    $ bin/omero chown 5 Dataset:53-56,65,101-105,201,202
+
+.. note::
+    When transferring multiple objects in a single command,
+    if one object cannot be transferred then the whole command will fail
+    and none of the specified objects will be transferred.
+    The :option:`chown --dry-run` option can be useful
+    as a check before trying to move large numbers of objects.
+
+Transferring lower level objects
+================================
+
+To transfer objects below a specified top-level object the following form
+of the object specifier is used.
+::
+
+    $ bin/omero chown 5 Project/Dataset/Image:51
+
+Here the all of images under the project 51 would be transferred. It is not
+necessary to specify intermediate objects in the hierarchy and so::
+
+    $ bin/omero chown 5 Project/Image:51
+
+would have the same effect as the call above.
+
+Transferring lower level objects
+================================
+
+Note that this feature is advanced and might be potentially slow.
+To transfer ownership of all objects belonging to a user or group of users
+the following form of the user specifier is used.
+::
+
+    $ bin/omero chown 10 Experimenter:1,3,7
+
+Here ownership of all the objects belonging to users 1,3 and 7
+would be transferred to user 10.
+
+Including and excluding objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. program:: chown
```

Comment:
This should probably be `.. program:: omero chown` else it seems we are documenting the system level `chown` command